src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -64,7 +64,7 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
             logger.WriteLine(Resources.Package_Initializing);
 
-            sseSessionManager = componentModel.GetService<ISSESessionManager>();
+            LoadServicesAndDoInitialUpdates(componentModel);
```

Comment:
I moved it up as on line 85 there is an await and this way it doesnt disturb the load order as much